### PR TITLE
Fix build action

### LINKED ISSUE #505: Fix build action
**Describe the bug**
Whenever we push commits, a [build action](https://github.com/cambiatus/frontend/blob/master/.github/workflows/on-push-action.yml) runs. Right now, it's failing with the following error:
```sh
Error: Cannot find module 'react-dev-utils/InterpolateHtmlPlugin'
```

**To Reproduce**
To reproduce, you can do the same as the action does:
1. Clone the repo
2. Run `yarn install --production`
3. Run `yarn build`

**Expected behavior**
The build action should complete successfully

**Screenshots**
2021-04-13-231609_930x848_scrot

**Desktop (please complete the following information):**
 - OS: Linux
 
**Additional context**
It works fine when building locally or for staging/demo/prod because we just use `yarn install`. The `--production` flag makes it so [we don't install devDependencies](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-production-true-false), and `react-dev-utils` is listed as a devDependency

### LINKED ISSUE #506: Fix Netlify deploys
**Describe the bug**
If you try to login on a Netlify deploy, you will get an "Authentication failed" error

**To Reproduce**
1. Go into a Netlify deploy (you can see a list of them [here](https://app.netlify.com/sites/eager-jepsen-ee4ac5/deploys) (the latest ones, from the support-subdomain branch might not even load):
2. Try to log in with a staging account
3. See the "Authentication failed" error

**Expected behavior**
You should be able to login as normal

**Screenshots**
2021-04-13-233250_1152x768_scrot

**Cambiatus Env you are using**
Netlify deploys/staging

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Browser Chromium, Firefox

**Additional context**
After #485, we introduced the `GRAPHQL_SECRET` environment variable, which is needed for the user to log in. We don't have that setup in Netlify, so that's why we can't login.

[This page](https://docs.netlify.com/configure-builds/environment-variables/) might help (haven't looked too much at it though)

### PR DESCRIPTION
**Pull request by NeoVier**
## What issue does this PR close
Closes #505, Closes #506

## Changes Proposed ( a list of new changes introduced by this PR)
- Move `react-dev-utils` from `devDependencies` to `dependencies`
- Add `.nvmrc` to specify node version on netlify deploys
- Add `netlify.toml` to add some netlify configurations (specially the build destination and the `GRAPHQL_SECRET` env variable)

## How to test ( a list of instructions on how to test this PR)
See if the build action completed successfully.

Additionally, if you want to try it out locally, you can clone the repo, switch to this branch, remove `node_modules` if you already have it, run `yarn install --production && yarn build`, which should complete successfully

### CODE REVIEW

**Comment by NeoVier** [comment]
Hey @lucca65 one of the build actions is failing because of sentry integration (you can see the error [here](https://github.com/cambiatus/frontend/pull/515/checks?check_run_id=2472049154)). Basically, we're missing some environment variables: `SENTRY_AUTH_TOKEN`, `SENTRY_ORG` and `SENTRY_PROJECT`. I can fill in `SENTRY_ORG` and `SENTRY_PROJECT`, but a manager needs to get the `SENTRY_AUTH_TOKEN` from their website ([here's how](https://github.com/marketplace/actions/sentry-release#create-a-sentry-internal-integration)). So can you setup `SENTRY_AUTH_TOKEN` in the [repo's secrets](https://github.com/cambiatus/frontend/settings/secrets/actions) please? I can setup the other two and include all of these in the workflow file

Not sure about netlify failing due to node version :thinking: (you can see the logs [here](https://app.netlify.com/sites/angry-wing-8fcdd0/deploys/608b5c8e05812d00081b72b1)). Do we have access to their dashboard/project settings?

**Comment by lucca65** [comment]
done @NeoVier! Thanks for the instructions made it easier and quick to get! 

Captura de Tela 2021-04-30 às 09 10 35

**Comment by NeoVier** [comment]
@lucca65 huh, that didn't work :thinking: 

It solved the error we were getting, but now we get an [Invalid token (http status: 401)](https://github.com/cambiatus/frontend/pull/515/checks?check_run_id=2475679842) error. I couldn't find much about this, any ideas?

**Comment by lucca65** [comment]
weird.. I've created a new one, please try again! 💪

**Comment by NeoVier** [comment]
Sentry worked now! Thanks! :muscle: 

Just need to fix netlify now... :thinking:

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+14.8.0
```
Comment (lucca65) [inline .nvmrc:1]: nvm could just read package.json 😔

**Review by lucca65 — APPROVED** [review]

File: .nvmrc (hunk 1 above)
Comment (NeoVier) [reply .nvmrc:1]: Yeah, that's pretty annoying

**Pull request merged**